mysql-common/usr/libexec/cont-setup.tpl

```diff
@@ -14,11 +14,13 @@ mysql_get_correct_config() {
 
     for f in `mysql_get_config_files_scl` ; do
         echo "$f"
-    done | grep -v mysql/my.cnf
+    done | grep -v mysql/my.cnf | head -n 1
```

Comment:
I've changed it a bit to be more robust and also add more comments. hopefully it seems better now.